packages/core-data/src/selectors.js

```diff
@@ -235,6 +235,7 @@ export const __experimentalGetDirtyEntityRecords = createSelector(
 								: entity.getTitle( entityRecord ),
 							name,
 							kind,
+							label: entity.label || '',
```

Comment:
@epiqueras -  Do you think its ok to add the `label` attribute to this `__experimentalGetDirtyEntityRecords` ?

It allows us to not have to useSelect for a redundant `getEntity` call here:
https://github.com/WordPress/gutenberg/blob/66792194130bbccad8fdae7ee36b938726ed3be9/packages/editor/src/components/entities-saved-states/index.js#L105-L109

I thought it would also help with generating the phrase but that did not end up being the case so we could go either way.

Reply:
That select only runs for the first entity in each group, yet here we add a label to every record.

The scale of this means the performance difference there is negligible, but maybe it's better if we don't assume all entities have labels. That selector could be used in another context where a different type of entity without labels is dirty.

Reply:
That makes sense to me.  That change has been reverted.